Sources/DateTime/InternalUTCClock.swift

```diff
@@ -31,33 +31,68 @@ public struct InternalUTCClock {
         }
 
         public init(secondsComponent: Int64, attosecondsComponent: Int64) {
-            self._value = Swift.Duration(secondsComponent: secondsComponent, attosecondsComponent: attosecondsComponent)
+            _value = Swift.Duration(secondsComponent: secondsComponent, attosecondsComponent: attosecondsComponent)
         }
 
         @available(*, deprecated, renamed: "seconds")
         public func secondsSinceEpoch() -> Int64 {
-            return self._value.components.seconds
+            _value.components.seconds
         }
 
         public func seconds() -> Int64 {
-            return self._value.components.seconds
+            _value.components.seconds
         }
-        
+
         public func attoseconds() -> Int64 {
-            return self._value.components.attoseconds
+            _value.components.attoseconds
         }
     }
 
     public init() {}
 }
 
+private extension String {
+    func pad(_ padding: Int = 2) -> String {
+        let toPad = padding - count
+        if toPad < 1 {
+            return self
+        }
+        return "".padding(toLength: toPad, withPad: "0", startingAt: 0) + self
```

Comment:
```suggestion
        return self.padding(toLength: toPad, withPad: "0", startingAt: 0)
```
?

Reply:
No, then you get trailing zeroes, if the number is "1", we want "01", not "10"